### PR TITLE
PT-1569 | update and clean up README

### PR DESCRIPTION
**Pull request by karisal-anders**
### docs: update and clean up README 

 - Remove beta from Kultus API as it is in production already
 - Add production environment's Kultus API URL
 - Add info on how to obtain LinkedEvents configuration for testing
 - Fix indentations
 - Don't start text rows with a comma or a dot character
 - Remove unnecessary URL namings where the plain URL works fine and
   can't go out of sync with the name which is almost the same as the
   URL
 - Normalize LE / LinkedEvent / LinkedEvents text usage in README to
   consistently use LinkedEvents

refs PT-1569 (setting up palvelutarjotin & palvelutarjotin-admin)

## Related

[PT-1569](https://helsinkisolutionoffice.atlassian.net/browse/PT-1569)

[PT-1569]: https://helsinkisolutionoffice.atlassian.net/browse/PT-1569?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**kultus-api branch is deployed to platta: https://kultus-api-pr300.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**kultus-api branch is deployed to platta: https://kultus-api-pr300.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**kultus-api branch is deployed to platta: https://kultus-api-pr300.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**kultus-api branch is deployed to platta: https://kultus-api-pr300.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**kultus-api branch is deployed to platta: https://kultus-api-pr300.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**kultus-api branch is deployed to platta: https://kultus-api-pr300.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**kultus-api branch is deployed to platta: https://kultus-api-pr300.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**kultus-api branch is deployed to platta: https://kultus-api-pr300.dev.hel.ninja** :rocket::rocket::rocket:

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 Production environment:
 
-- https://api.hel.fi/kultus-beta/graphql
+- https://palvelutarjotin-api.prod.kuva.hel.ninja/graphql
```
Comment (nikomakela) [inline README.md:10]: Should we already mention the new Platta address also?

**Review by nikomakela — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (karisal-anders) [reply README.md]: Added Platta addresses and status of the use of the KuVa and Platta addresses on 2023-02-24 to README and force pushed the [changes](https://github.com/City-of-Helsinki/palvelutarjotin/compare/bd3adb622ff4aabe50ca6c060629e28d4ecff15f..b129018ac993c1a6de58e4f49ba68e11f9f58ebc).

**Comment by terovirtanen** [comment]
**kultus-api branch is deployed to platta: https://kultus-api-pr300.dev.hel.ninja** :rocket::rocket::rocket:

**Pull request merged**